src/js/components/Drop/stories/Inline.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import { Box, Drop } from 'grommet';
+
+const align = { top: 'bottom', left: 'left' };
+
+const InlineDrop = () => {
+  const targetRef = useRef();
+
+  // trigger re-render so we have the targetRef
+  const [, setShowDrop] = useState(false);
+  useEffect(() => {
+    setShowDrop(true);
+  }, []);
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Box fill align="center" justify="center">
+      <Box
+        background="dark-3"
+        pad="medium"
+        align="center"
+        justify="start"
+        ref={targetRef}
+      >
+        Target
+        {targetRef.current && (
+          <Drop container="inline" align={align} target={targetRef.current}>
```

Comment:
In this story example, the `container` is the prop and 'inline' is the value. Is this the expected usage? 
There are no details about a `container` prop. 
The prop `inline` should be a boolean according to the specified types.